### DISCUSSION TITLE
"Move to a new file" not always working in TS

### DISCUSSION THREAD

**Discussion by NigelThorne** [zed-industries/zed#17289] [Feature Requests]
How can I debug why "move to a new file" isn't doing anything. I'm working in Typescript.

**Comment by vincehi** [comment]
Solution : https://github.com/zed-industries/zed/discussions/21565

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
